core/ocaml_exception_info.mli

```diff
@@ -0,0 +1,18 @@
+type t
```

Comment:
This module could use a comment. I had some pretty basic questions when I first read it like "what is a pushtrap?"

Reply:
Done 👍